public/index.php

```diff
@@ -24,7 +24,7 @@
 // Setup autoloading
 include 'vendor/autoload.php';
 
-if (!defined('APPLICATION_PATH')) {
+if (! defined('APPLICATION_PATH')) {
     define('APPLICATION_PATH', realpath(__DIR__ . '/../'));
```

Comment:
Do we need `realpath` in this place?

Please review rest of this file, I think part for IBM i can be now removed from `index.php`